modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -303,15 +303,15 @@ export async function getBitgoToUserLatestShare(
  * @param keyShare - TSS key share of the party preparing exchange materials
  * @param recipientIndex - index of the recipient (1, 2, or 3)
  * @param recipientGpgPublicArmor - recipient's public gpg key in armor format
- * @param gpgKey - ephemeral GPG key to encrypt / decrypt sensitve data exchanged between user and server
+ * @param senderGpgKey - ephemeral GPG key to encrypt / decrypt sensitve data exchanged between user and server
  * @param isbs58Encoded - is bs58 encoded or not
  * @returns encrypted N Share
  */
 export async function encryptNShare(
   keyShare: KeyShare,
   recipientIndex: number,
   recipientGpgPublicArmor: string,
-  gpgKey: pgp.SerializedKeyPair<string>,
+  senderGpgKey: pgp.SerializedKeyPair<string>,
```

Comment:
breaking name changes need to have `BREAKING: ` in the commit message. i.e. https://github.com/BitGo/BitGoJS/pull/2565

Reply:
is this really a breaking change? it's not a typed params so whatever people passing in the original order will still be the same order and nothing will change

Reply:
ah, fair enough, my bad